fortress/ros_gz_project_template_guide.md

```diff
@@ -32,15 +34,53 @@ In this guide, you will learn how to use the `ros_gz_project_template` to create
-Setting up paths can be also automated using colcon environment hooks with a [DSV file](https://colcon.readthedocs.io/en/released/developer/environment.html?highlight=dsv#dsv-files) prepending the model share path to Gazebo resource path.
+* `ros_gz_example_application` - holds ROS 2 specific code and configurations. Namely where control, planning or any high level algoritms reside.
+
+   <pre> ├── CMakeLists.txt
+   ├── package.xml
+   ├── <span style="color:#12488B"><b>src</b></span>
+       └── ...
+   </pre>
+
+* `ros_gz_example_bringup` - holds launch files and high level utilities, communication bridge between ROS and Gazebo. Any robot or hardware specific configurations go here.
+
+   <pre> ├── <span style="color:#12488B"><b>config</b></span>
+   │   ├── ros_gz_example_bridge.yaml
+   │   └── diff_drive.rviz
+   ├── <span style="color:#12488B"><b>launch</b></span>
+       └── diff_drive.launch.py
+   </pre>
+
+* `ros_gz_example_description` - holds the SDF description of the simulated system and any other [simulation assets](#accessing-simulation-assets).
+
+   <pre> ├── <span style="color:#12488B"><b>hooks</b></span>
+   │   └── ros_gz_example_description.dsv.in
+   ├── <span style="color:#12488B"><b>models</b></span>
+       ├── <span style="color:#12488B"><b>diff_drive</b></span>
+           ├── model.config
+           └── model.sdf
+   </pre>
 
 * `ros_gz_example_gazebo` - holds Gazebo specific code and configurations. Namely this is where user-defined worlds and custom system plugins end up.
 
-* `ros_gz_example_application` - holds ROS 2 specific code and configurations. Namely where control, planning or any high level algoritms reside.
+   <pre> ├── <span style="color:#12488B"><b>include</b></span>
+   │   └── <span style="color:#12488B"><b>ros_gz_example_gazebo</b></span>
+   │       ├── BasicSystem.hh
+   │       └── FullSystem.hh
+   ├── <span style="color:#12488B"><b>src</b></span>
+   │   ├── BasicSystem.cc
+   │   └── FullSystem.cc
+   ├── <span style="color:#12488B"><b>worlds</b></span>
+           └── diff_drive.sdf
+   </pre>
 
-* `ros_gz_example_bringup` - holds launch files and high level utilities, communication bridge between ROS and Gazebo. Any robot or hardware specific configurations go here.
+## Accessing Simulation Assets
+
+Simulation assets include your models or robot descriptions in URDF or SDF, meshes and materials files to help visualize different parts of the robot and finally compiling all these elements in a simulated world SDF. Gazebo offers a few different mechanisms for locating those, initializing it's search on `IGN_GAZEBO_RESOURCE_PATH` environment variable, see ign-gazebo API on [finding resources](https://gazebosim.org/api/sim/6/resources.html) for more details.
```

Comment:
I know Fortress is still ign but maybe it worth it to use th right name. Just a suggestion
ign-gazebo to gz-sim ?

```suggestion
Simulation assets include your models or robot descriptions in URDF or SDF, meshes and materials files to help visualize different parts of the robot and finally compiling all these elements in a simulated world SDF. Gazebo offers a few different mechanisms for locating those, initializing it's search on `IGN_GAZEBO_RESOURCE_PATH` environment variable, see gz-sim API on [finding resources](https://gazebosim.org/api/sim/6/resources.html) for more details.
```

Reply:
Tbh I was on the edge too, going for using `gz-sim`